libp2p/protocol_muxer/multiselect.py

```diff
@@ -48,7 +48,6 @@ def add_handler(
         """
         self.handlers[protocol] = handler
 
-    # FIXME: Make TProtocol Optional[TProtocol] to keep types consistent
```

Comment:
You've removed the `FIXME` comment, but not updated the return type accordingly.